codeframe/core/models.py

```
@@ -210,4 +210,155 @@ class BlockerListResponse(BaseModel):
+    last_accessed: datetime
+
+
+class ContextItemCreateModel(BaseModel):
+    """Request model for creating a context item."""
+    item_type: ContextItemType
+    content: str = Field(..., min_length=1, max_length=100000)
+
+    def validate_content(self) -> str:
+        """Validate content is not empty or whitespace-only."""
+        if not self.content.strip():
+            raise ValueError("Content cannot be empty or whitespace-only")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)